client/components/jetpack/licensing-activation-banner/index.tsx

```diff
@@ -12,19 +14,36 @@ interface Props {
 
 function LicensingActivationBanner( { siteId }: Props ) {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
 	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
 	const jetpackDashboardUrl = siteAdminUrl + 'admin.php?page=jetpack#/license/activation';
 	const userLicensesCounts = useSelector( getUserLicensesCounts );
 	const hasDetachedLicenses = userLicensesCounts && userLicensesCounts[ 'detached' ] !== 0;
 
+	useEffect( () => {
+		dispatch(
+			recordTracksEvent( 'calypso_jetpack_licensing_activation_banner_render', { site_id: siteId } )
+		);
+	}, [ dispatch, siteId ] );
```

Comment:
I think this should have no dependencies to make it run only on the first render.
```suggestion
		// eslint-disable-next-line react-hooks/exhaustive-deps
	}, [] );
```